business_requirement_deliverable/models/business.py

```diff
@@ -78,8 +78,6 @@ def product_id_change(self):
             uom_id = product.uom_id.id
         if product.description_sale:
             description += '\n' + product.description_sale
-        if not self.name:
```

Comment:
That's what I thought, just wanted to get rid of the annoying travis failing for outside code. I'll revert and commit ASAP. Thanks again to all of you for your input.